wp-resources/themes/webonary-zeedisplay/style copy.css

```diff
@@ -1,1124 +1,1124 @@
-}
-.wp-pagenavi a:hover {
-  background: url(images/button_hover.png) repeat-x #eee;
-}
-.wp-pagenavi .current {
-    margin: 0 5px 0 0;
-    padding: 3px 8px;
-    background-color: #0066ee;
-    color: #fff;
-    text-align: center;
-  font-size: 1.1em;
-  font-weight: bold;
-}
-/*============================ BOTTOM BAR ============================*/
-#bottombar {
-  margin-top: 20px;
-  padding: 15px 10px;
-  background: #444;
-  border-top: 7px solid #666;
-}
-#bottombar ul li h2 {
-  color: #fff;
-  background: url(images/navi_bg.png) repeat-x #505050;
-  height: 30px;
-  line-height: 30px;
-  font-size: 1.1em;
-  font-weight: bold;
-  padding: 0 15px;
-  margin-bottom: 15px;
-  font-family: verdana, arial;
-}
-#bottombar ul li {
-  width: 198px;
-  float:left;
-  margin: 10px 5px;
-  padding: 5px;
-  border: 1px solid #505050;
-}
-#bottombar ul li div {
-  margin: 0;
-  padding: 0 5px;
-  line-height: 1.5em;
-}
-#bottombar ul li ul {
-  padding: 0 5px;
-  list-style: circle inside !important;
-  color: #ddd;
-}
-#bottombar a, #bottombar ul {
-  color:#fff;
-  font-size: 1.0em;
-  text-decoration: none;
-}
-#bottombar a:hover {
-  color:#aaa;
-  text-decoration: underline;
-}
-#bottombar ul ul li {
-  margin: 0;
-  padding: 3px;
-  border: none;
-}
-/*============================ FOOT ============================*/
-#footer {
-  clear: both;
-  margin: 0 0 10px 0;
-  padding: 3px 5px 3px 10px;
-  height: 100%;
-  line-height: 1.2em;  
-  font-family: tahoma, arial;
-  color: #fff;
-}
-
-.pink
-{
-  background: #85005B;    
-}
-.blue
-{
-  background: #0065A5;
-}
-#footer p {
-  font-size: 0.9em;
-  color: #555;
-  padding: 10px;
-  margin: 0;
-  text-align: left;
-}
-#footer span { color:#fff;}
-#footer div { color:#fff;}
-#footer sup { color:#fff;}
-
-#footer a {
-  color: #fff;
-}
-#footer a:hover {
-  text-decoration: none;
-}
-
-/*
- * The credit_link is used for the bottom right footer, and originally had a
- * link to the original theme site. It's included here to get the full font size.
- * A link to the theme author's site still exists (as of this date) in the
- * zeeDisplay theme options (theme-admin.php).
- */
-.credit_link {
-  float: right;
-  font-size: 1.0em;
-  margin: 0px 0px 0 0px;
-}
-
-/*============================ Webonary Styles ============================*/
-
-/* This particular theme displays the headword prominently. No need to repeat it.  */
-/* .headword {
-  display: none;
-}
-
-/*
- * The languages drop down box was too wide for a loooong language name.
- */
-.webonary_searchform_language_select {
-  width:250px;
-}
-
-/* Image wrapping in Wordpress menu pages */
-img.alignright {float:right; margin:0 0 1em 1em}
-img.alignleft {float:left; margin:0 1em 1em 0}
-img.aligncenter {display: block; margin-left: auto; margin-right: auto}
-.alignright {float:right; }
-.alignleft {float:left; }
-.aligncenter {display: block; margin-left: auto; margin-right: auto}
+/*======================================
```

Comment:
Shouldn't this file just be deleted?

Reply:
It probably should. It was added in 2015, part of the initial commit.
![image](https://user-images.githubusercontent.com/5262964/185111808-d0e080ca-908e-49d4-a97f-b0891238c640.png)